consumers/src/unit-test/java/uk/gov/moj/sdt/cmc/consumers/CMCConsumerGatewayTest.java

```diff
@@ -62,12 +77,32 @@ void shouldInvokeBreathingSpace() throws Exception {
         verify(individualRequest).setRequestStatus(ProcessingStatus.PROCESSED.name());
     }
 
-    private void setupMockBehaviour() {
-        BreathingSpaceRequest breathingSpaceRequest = mock(BreathingSpaceRequest.class);
-        when(individualRequest.getRequestPayload()).thenReturn(XML);
-        try {
+    @Test
+    void shouldInvokeClaimStatusUpdate() throws Exception {
+        setupMockBehaviour(CLAIM_STATUS_UPDATE);
+        ClaimStatusUpdateResponse response = new ClaimStatusUpdateResponse();
+        response.setProcessingStatus(ProcessingStatus.PROCESSED);
+        when(claimStatusUpdate.claimStatusUpdate(anyString(), anyString(), any())).thenReturn(response);
+
+        cmcConsumerGateway.individualRequest(individualRequest, CONNECTION_TIME_OUT, RECEIVE_TIME_OUT);
+
+        verify(claimStatusUpdate).claimStatusUpdate(anyString(), anyString(), any(ClaimStatusUpdateRequest.class));
+        verify(xmlToObject).convertXmlToObject(anyString(), any());
+        verify(individualRequest).getRequestPayload();
+    }
+
+    private void setupMockBehaviour(String requestType) throws Exception {
+        when(individualRequest.getSdtRequestReference()).thenReturn(SDT_REFERENCE);
+        if (requestType.equals(BREATHING_SPACE)) {
+            BreathingSpaceRequest breathingSpaceRequest = mock(BreathingSpaceRequest.class);
+            when(individualRequest.getRequestPayload()).thenReturn(XML);
```

Comment:
If this setup is going to be common to each type then have one instance of it outside of the if-elseif statement.

Reply:
done